FLIR/conservator/util.py

```diff
@@ -53,3 +55,16 @@ def base_convert(b, n):
         output.append(r)
         n = int(n / b)
     return output
+
+
+def chunks(list, size):
+    """
+    Splits a :param list: into lists of :param size:.
+    Returns an iterator of these lists.
+    Once the input list is exhausted, the last list in the output
+    iterator will be padded by None's to make up the size difference.
+    e.g. chunks(['a', 'b', 'c'], 2) will return an iterator containing
+    ['a', 'b'] and ['c', None]
+    Adapted from  https://stackoverflow.com/a/312644
+    """
```

Comment:
Mostly, for method parameter docstrings we have followed the sphinx docstring formatting described here for the automated documentation generator: https://thomas-cokelaer.info/tutorials/sphinx/docstring_python.html#template.MainClass1 -- except for the :type param: and :rtype: lines, which for conservator-cli API methods should be replaced with Python type hinting described in PEP484 (https://peps.python.org/pep-0484/).  The type hinting is new (for us), but ideally at least externally visible methods/functions should have them.